cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert.hs

```diff
@@ -1390,16 +1390,15 @@ insertScript ::
   Trace IO Text ->
   DB.TxId ->
   Generic.TxScript ->
-  ExceptT SyncNodeError (ReaderT SqlBackend m) DB.ScriptId
+  ReaderT SqlBackend m DB.ScriptId
```

Comment:
How come we're removing the error handling `SyncNodeError`?

Reply:
`SyncNodeError` cannot happen in this function. `ReaderT` is lifted to `ExceptT`, but that's unnecessary.